doc/amazon_sagemaker_debugger.rst

```diff
@@ -0,0 +1,377 @@
+
+    collection_config_1 = CollectionConfig(
+        name='collection_name_1',
+        parameters={
+            'include_regex': '.*'
+        }
+    )
+    collection_config_2 = CollectionConfig(
+        name='collection_name_2',
+        parameters={
+            'include_regex': '.*'
+        }
+    }
+
+    debugger_hook_config = DebuggerHookConfig(
+        s3_output_path='s3://path/for/data/emission',
+        container_local_output_path='/local/path/for/data/emission',
+        hook_parameters={
+            'save_interval': '10'
+        },
+        collection_configs=[
+            collection_config_1, collection_config_2
+        ]
+    )
+
+In the above sample code, the ``save_interval`` of ``10`` will be applied for storing both collections.
+
+Note that the value set in the ``collection_parameters`` for a parameter will override the corresponding value of the same parameter in the ``hook_parameters``. For example, in the above sample code, if ``collection_config_2`` had the value of ``save_interval`` set something other than ``10``, say, ``20``, then the tensors for that collection would have been saved with step interval ``20`` while those for ``collection_config_1`` would still be saved with ``10``.
+
+Begin model training
+--------------------
+
+To create a training job that initializes the debugging hook with the value of the ``DebuggerHookConfig`` object, call ``fit()`` on the ``estimator``. The hook starts emitting the relevant debugging data, i.e. the tensor collections, in real time and stores the data locally in the local path provided in ``DebuggerHookConfig``. This data is then uploaded in near real time to an S3 path derived from the path provided in the hook configuration.
+
+.. code::
+
+    s3://{destination-bucket-prefix}/{training-job-name}/debug-output/
+
+The path is derived from the value of ``s3_output_path`` and not used verbatim to ensure that artifacts from different training jobs are placed in different Amazon S3 paths (it's a good practice to keep the debug artifacts separate for different training jobs for correct analyses later on.)
+
+To access the above Amazon S3 path through the estimator object, you can do:
+
+.. code:: python
+
+    tensors_s3_output_path = estimator.latest_job_debugger_artifacts_path()
+
+
+You can use the ``S3Downloader`` utility to view and download the debugging data emitted during training in Amazon S3 (Note that data is stored in a streaming fashion so the data you download locally through ``S3Downloader`` will be a snapshot of the data generated until that time.):
+
+.. code:: python
+
+    from sagemaker.s3 import S3Downloader
+
+    # Start the training by calling fit
+    # Setting the wait to `False` would make the fit asynchronous
+    estimator.fit(wait=False)
+
+    # Get a list of S3 URIs
+    S3Downloader.list(estimator.latest_job_debugger_artifacts_path())
+
+On by default behavior and opting out
```

Comment:
I think this section should probably be its own, maybe at the end above "Learn More"

also the title is a little clunky - maybe just "Default behavior"? (perhaps @aws-rbs has a better suggestion)

Reply:
Agreed - it was feeling out of place there. Changed the section heading to "Default behavior" unless @aws-rbs has a better suggestion.